protected/views/patient/crud/_conflicts.php

```diff
@@ -6,7 +6,7 @@
  * Time: 12:01 PM
  */ ?>
 
-<div id="conflicts" class="large-6 column end">
+<div id="conflicts" class="large-4 column end">
```

Comment:
Will this affect the conflict popup?

Reply:
It will, but that's the idea. Doing this will move it back to its original location, albeit smaller than before.